crates/router/src/connector/utils.rs

```diff
@@ -510,6 +511,10 @@ impl PaymentsAuthorizeRequestData for types::PaymentsAuthorizeData {
             .as_ref()
             .map(|surcharge_details| surcharge_details.get_total_surcharge_amount())
     }
+
+    fn is_merchant_initiated_mandate_payment(&self) -> bool {
+        self.setup_mandate_details.is_some()
```

Comment:
setup_mandate_details will be present only incase of CIT not MIT

Reply:
will update it